ste/mgr-JobPartMgr.go

```diff
@@ -460,44 +461,49 @@ func (jpm *jobPartMgr) clientInfo() {
 		Cancel:   jpm.jobMgr.Cancel,
 	}
 
-	retryOptions := policy.RetryOptions{
-		MaxRetries:    UploadMaxTries,
-		TryTimeout:    UploadTryTimeout,
-		RetryDelay:    UploadRetryDelay,
-		MaxRetryDelay: UploadMaxRetryDelay,
-	}
+	/*
```

Comment:
let's just go ahead and delete this code, it will be in the github history anyway if we need to go back and reference

Reply:
There is lot more dead code. I'll clean up either for this release or next. Particularly this file